src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionProxy.cs

```diff
@@ -2169,6 +2168,7 @@ public CimSessionProxyNewCimInstance(string computerName, CimNewCimInstance oper
         /// <summary>
         /// Initializes a new instance of the <see cref="CimSessionProxyNewCimInstance"/> class.
         /// </summary>
+        /// <param name="computerName"></param>
```

Comment:
This method doesn't have a `computerName` param, that's the one above and the second param is missing?